plugins/main/public/components/wz-blank-screen/wz-blank-screen.js

```diff
@@ -16,19 +16,73 @@ import {
   PLUGIN_PLATFORM_WAZUH_DOCUMENTATION_URL_PATH_TROUBLESHOOTING,
   PLUGIN_PLATFORM_URL_GUIDE,
   PLUGIN_PLATFORM_URL_GUIDE_TITLE,
+  UI_LOGGER_LEVELS,
 } from '../../../common/constants';
 import { webDocumentationLink } from '../../../common/services/web_documentation';
+import { AppState } from '../../react-services/app-state';
+import { WzMisc } from '../../factories/misc';
+import { getAngularModule } from '../../kibana-services';
+import { ErrorHandler } from '../../react-services/error-handler';
+import { UI_ERROR_SEVERITIES } from '../../react-services/error-orchestrator/types';
+import { getErrorOrchestrator } from '../../react-services/common-services';
 
 export class WzBlankScreen extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      errorToShow: null,
+    };
+
+    // Get $location forom AngularJS
+    const $injector = getAngularModule().$injector;
```

Comment:
The AngularJS dependecies must be removed.

Reply:
See https://github.com/wazuh/wazuh-dashboard-plugins/pull/6538#discussion_r1551093623